packages/xo-web/src/common/intl/messages.js

```diff
@@ -1051,12 +1051,12 @@ const messages = {
   importVdi: 'Import VDI content',
   importVdiNoFile: 'No file selected',
   selectVdiMessage: 'Drop VHD file here',
-  srsNotOnSameHost:
-    'The SRs must either be shared or on the same host for the VM to be able to start.',
   useQuotaWarning:
     'Creating this disk will use the disk space quota from the resource set {resourceSet} ({spaceLeft} left)',
   notEnoughSpaceInResourceSet:
     'Not enough space in resource set {resourceSet} ({spaceLeft} left)',
+  warningSr:
+    'The SR{n, plural, one {} other {s}}  must either be shared or on the same host for the VM to be able to start.',
```

Comment:
Always plural. There's no point of saying this for a single SR.

Also, you can say "The VDIs' SRs" actually...